plugins/spu2-x/src/Linux/Config.cpp

```diff
@@ -174,6 +174,12 @@ void ReadSettings()
 
     Clampify(SndOutLatencyMS, LATENCY_MIN, LATENCY_MAX);
 
+    if (mods[OutputModule] == NULL) {
+        fwprintf(stderr, L"* SPU2-X: Unknown output module '%s' specified in configuration file.\n", temp);
```

Comment:
Clang errored out on this line. Looking closer, temp is a wxString, so would need to be converted. I believe it'd be temp.c_str() .

Reply:
Oops, while clang does not build here, it gets far enough to have caught this. However `c_str()` does not work, but looking at the 3rdparty wx code it seems `wc_str()` does. I updated the PR with this change, does this look better to you?